coalib/bearlib/languages/definitions/ZShell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
```

Comment:
Please remove this file . And read your PR diff after uploading it to find such obvious mistakes before wasting reviewer time.

Reply:
Correct me if I am wrong but I suppose I have added certain command names in ZShell file right? They too were reserved words and there was no concept of a floating point number in bash,so I added float to the zshell keywords.